aiidalab_widgets_base/computational_resources.py

```diff
@@ -1228,21 +1215,16 @@ def refresh(self, _=None):
 
     @traitlets.validate("value")
     def _validate_value(self, change):
-        """Select computer either by label or by class instance."""
-        computer = change["value"]
+        """Select computer by computer UUID."""
+        computer_uuid = change["value"]
         self.output.value = ""
-        if not computer:
-            return None
-        if isinstance(computer, str):
-            if computer in self.computers:
-                return self.computers[computer]
-            self.output.value = f"""Computer instance '<span style="color:red">{computer}</span>'
-            is not configured in your AiiDA profile."""
+        if not computer_uuid:
             return None
 
-        if isinstance(computer, orm.Computer):
-            if computer.label in self.computers:
-                return computer
-            self.output.value = f"""Computer instance '<span style="color:red">{computer.label}</span>'
-            is not configured in your AiiDA profile."""
-        return None
+        try:
+            _ = UUID(computer_uuid, version=4)
```

Comment:
Does this work also if `computer_uuid is None`? There was an explicit check for this previously.

Reply:
`computer_uuid is None` is included by `if not computer_uuid:` (two lines ahead), correct?